src/pudl/analysis/plant_parts_eia.py

```diff
@@ -240,6 +246,8 @@
     "plant_technology",
     "plant_prime_fuel",
     "plant_ferc_acct",
+    "plant_installation_year",
+    "plant_construction_year",
```

Comment:
These belong in `PLANT_PARTS_ORDERED`, right? I wasn't entirely sure if they also should go in other globals like `FIRST_COLS`.